x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/aggregation/ArrayStateTests.java

```diff
@@ -54,8 +79,9 @@ public ArrayStateTests(DataType type, int valueCount, boolean inOrder) {
             case IP -> ElementType.BYTES_REF;
             default -> throw new IllegalArgumentException();
         };
-        this.valueCount = valueCount;
+        this.valueCount = valueCount.getAsInt();
         this.inOrder = inOrder;
+        logger.info("value count is {}", this.valueCount);
```

Comment:
To be removed? It won't flood the logs, but not sure if it's intended

Reply:
It's intended though not super important. If the parameter is random it's nice to have the real value written down.